techtv2ovs/utils.py

```diff
@@ -0,0 +1,302 @@
+""" techtv2ovs utils """
+import logging
+import re
+import sys
+from html import unescape
+from unicodedata import normalize
+
+import MySQLdb
+import boto3
+import bonobo
+
+from django.contrib.auth.models import User
+from django.conf import settings
+from django.utils.html import strip_tags
+
+from techtv2ovs.constants import TTV_THUMB_BUCKET, TTV_VIDEO_BUCKET, ImportStatus
+from techtv2ovs.models import TechTVCollection, TechTVVideo
+from techtv2ovs.tasks import process_videofiles
+
+from ui.models import Collection, MoiraList, Video, VideoThumbnail
+
+# pylint: disable=attribute-defined-outside-init,broad-except,too-many-instance-attributes
+
+
+def mysql_query(query, connection):
```

Comment:
Can we update this to support parameterized queries instead of using string formatting?

Reply:
✅